src/Sarif/Visitors/SarifCurrentToVersionOneVisitor.cs

```diff
@@ -204,6 +206,43 @@ internal PhysicalLocationVersionOne CreatePhysicalLocation(PhysicalLocation v2Ph
             return physicalLocation;
         }
 
+        internal RegionVersionOne CreateRegion(Region v2Region)
+        {
+            RegionVersionOne region = null;
+
+            if (v2Region != null)
+            {
+                region = CreateRegion(v2Region.StartColumn,
+                                      v2Region.StartLine,
+                                      v2Region.EndColumn,
+                                      v2Region.EndLine,
+                                      v2Region.Length,
+                                      v2Region.Offset);
+            }
+
+            return region;
+        }
+
+        internal RegionVersionOne CreateRegion(int startColumn, int startLine, int endColumn = 0, int endLine = 0, int length = 0, int offset = 0)
```

Comment:
Why do you need this overload?

Reply:
Copied from the other transformer, where I do need it. Not needed here yet, will remove.